README.md

```diff
@@ -1,2 +1,11 @@
-# CircleCI-HockeyApp-Appium-ReactNative
-Guidance on integrating Kobiton service into the mobile app build pipeline: CircleCI, HockeyApp, Appium and ReactNative
+# Integrating Kobiton into CircleCI-HockeyApp-Appium-ReactNative development pipeline
+
+Kobiton is a mobile **experience** platform that accelerates delivery and testing of mobile apps by offering manual and automated testing on real devices, in the cloud or on-premise.
+
+By integrating Kobiton into mobile application development pipeline, developers can be able to execute manual and automation tests on a wide range of real devices, thus can greatly improve the quality of their applications.
```

Comment:
1. apply automation testing
2. Using source code control for saving test script.